### PR TITLE
add eradisplay to stage 0 of ECMA402

### PR DESCRIPTION
**Pull request by FrankYFTang**
@sffc @Louis-Aime @gibson042 @leobalter @zbraniecki

### CODE REVIEW

**Review by zbraniecki — APPROVED** [review]

**Review by septs — APPROVED** [review]

File: ecma402/stage-0-proposals.md [hunk 1]
```diff
@@ -9,9 +9,11 @@ Stage 0 proposals are either
 | -------------------------------------------------------------------- | ---------------- | ------------------ |
 | [Fix 9.2.3 LookupMatcher algorithm][]                                | Rafael Xavier    | Rafael Xavier      |
 | [`Intl.NumberFormat` `round` option][intl.numberformat round option] | Rafael Xavier    | Rafael Xavier      |
+| [eraDisplay option for Intl.DateTimeFormat][eradisplay]              | Louis-Aime       | Louis-Aime         |
```
Comment (septs) [inline ecma402/stage-0-proposals.md:12]:
```suggestion
| [eraDisplay option for `Intl.DateTimeFormat`][eradisplay]              | Louis-Aime       | Louis-Aime         |
```

**Comment by ljharb** [comment]
Which meeting agenda is this stage 0 proposal on?

**Comment by sffc** [comment]
We discussed it at the TC39-TG2 meeting last month ([notes](https://github.com/tc39/ecma402/blob/master/meetings/notes-2020-11-05.md#option-eradisplay-in-intldatetimeformat-426)) and in https://github.com/tc39/ecma402/issues/426#issuecomment-722726223.  Presumably it will go on the TC39-TG1 meeting agenda in January.

**Comment by ljharb** [comment]
Perfect, thanks :-)

**Pull request merged**